pkg/storage/chunk/fetcher/fetcher.go

```diff
@@ -186,6 +185,24 @@ func (c *Fetcher) FetchChunks(ctx context.Context, chunks []chunk.Chunk, keys []
 	log := spanlogger.FromContext(ctx)
 	defer log.Span.Finish()
 
+	// Extend the extendedHandoff to be 10% larger to allow for some overlap becasue this is a sliding window
+	// and the l1 cache may be oversized enough to allow for some extra chunks
+	extendedHandoff := c.l2CacheHandoff + (c.l2CacheHandoff / 10)
```

Comment:
Shouldn't we take `max_chunk_age` into account here? That'd seem more idiomatic than a 10% increase here to me.

Reply:
interesting. thinking about this a bit I'm not sure that max_chunk_age really would be the right pick either. 

this "fudge factor" really exists just to allow for some additional cache hits as the handoff duration rolls through time and the L1 cache has items that could be hits. If it hits or not is really a function of if the L1 cache is sized properly.

How old a chunk can get with max_chunk_age doesn't seem really related to this to me either, another way of thinking about it is changing max_chunk_age wouldn't really have any affect on this fudge factor.

The 10% is totally arbitrary, not sure though what we could do here that isn't arbitrary.

I'm inclined to leave it like this for now and maybe we'll find ourselves back here in the future if there is an optimization or unrealized error in this approach.